modules/sdk-coin-trx/test/unit/trx.ts

```diff
@@ -421,6 +421,7 @@ describe('TRON:', function () {
         userKey: TestRecoverData.userKey,
         backupKey: TestRecoverData.backupKey,
         bitgoKey: TestRecoverData.bitgoKey,
+        tokenContractAddress: 'TSdZwNqpHofzP6BsBKGQUWdBeJphLmF6id',
```

Comment:
Can you add tokenContractAddress validation in any recoverConsolidations testcase.